src/btree/bt_vrfy_dsk.c

```diff
@@ -341,7 +359,7 @@ __verify_row_key_order_check(WT_SESSION_IMPL *session, WT_ITEM *last, uint32_t l
     WT_ERR(__wt_scr_alloc(session, 0, &tmp2));
 
     ret = WT_ERROR;
-    WT_ERR_VRFY(session,
+    WT_ERR_VRFY(session, vi->flags,
```

Comment:
We have two sessions here. The session passed in and the session in vi. They are probably the same but feels duplicate to me.

Reply:
Thanks for picking that up, I must have missed refactoring that function. I can definitely refactor this function but feels like we should first decide if we want to pass session around separately to the `vi` struct. @luke-pearson do you have any opinion on this?

Reply:
Chenhao is correct that we generally pass the session as an arg followed by other args. I think I could go either way but for consistency probably pull the session out.

Reply:
I agree that we often pass a session around directly, but that's because it is used as the way of passing the broader context around. In this case, I prefer to keep the number of parameters smaller, and pass around the context structure that contains the session handle.

Think of WT_VERIFY_INFO like a C++ class associated with the verify functions, the session would be a member variable.

See the WT_DBG structure for an example: https://github.com/wiredtiger/wiredtiger/blame/develop/src/btree/bt_debug.c#L18